### DISCUSSION TITLE
关于iOS和Windows同步

### DISCUSSION THREAD

**Discussion by joyengh** [obsidianzh/forum#116] [Q&A常见问题]
iPad上在obsidian内创建了一个在iCloud上的库，但是Windows上的文件查看器找不这个文件夹（那个Obsidian是我自己加上去的，连图标都没有……），而且网页端的iCloud也看不到这个带obsidian图标文件夹

![image](https://user-images.githubusercontent.com/90150309/132135272-41be626f-9c0e-487b-b2bb-5cf3450682b7.png)
![image](https://user-images.githubusercontent.com/90150309/132135341-1e7c57e2-3520-4ec7-9f07-99213dd18f5e.png)
![image](https://user-images.githubusercontent.com/90150309/132135490-25b47e4b-d080-4320-8c43-edcdf18f312c.png)

各位大佬，求教

**Comment by chenchenxl** [comment]
要在windows 下安装 icloud ，并把文件放到icloud 下的obsidian文件里
